modules/system/oagw/docs/adr-oauth2-client-credentials-auth-plugin.md

```
@@ -0,0 +1,515 @@
+      "issuer_url": "https://accounts.google.com",
+      "client_id_ref": "cred://google-client-id",
+      "client_secret_ref": "cred://google-client-secret",
+      "scopes": "https://www.googleapis.com/auth/cloud-platform"
+    }
+  }
+}
+```
+
+### Config Fingerprint (Cache Key)
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)